app/main/views/direct_award.py

```diff
@@ -38,7 +43,7 @@ def list_projects():
     )
 
     return jsonify(
-        projects=[project.serialize() for project in projects.items],
+        projects=[project.serialize(with_users=with_users) for project in projects.items],
```

Comment:
I think project.users is not `lazy=joined`, so this will generate 100 additional queries to the database per page (assuming each project has only one user).

Even if this is meant to be used in scripts, we've seen similar requests time out in the past and a slow response will show up in our monitoring, which would make any alerting we build on top of it less useful.

I think we could use the query option setter here to set the join option on the initial query, the same way we disable it when we don't need it in other endpoints.

Reply:
What's the query option setter? Can you link me to an example?

Reply:
https://github.com/alphagov/digitalmarketplace-api/blob/d8a7ee8267ce1331d45b12e151f4b8c2f6307348/app/main/views/brief_responses.py#L219-L223

Reply:
Thanks, have updated to a joinedload for users. Is much quicker now!